user-interface/src/App.tsx

```diff
@@ -55,6 +56,7 @@ function App() {
             <Route path="/search" element={<SearchScreen />}></Route>
             <Route path="/search/:caseId" element={<SearchScreen />}></Route>
             <Route path="/case-assignment" element={<CaseAssignment />}></Route>
+            <Route path="/my-cases" element={<MyCasesScreen />}></Route>
```

Comment:
Should we make "My Cases" the new default screen? At some point the case assignment screen will be the CaseAssignmentManagers' "My Cases" screen.